### PR TITLE
Import custom c++ types as unchecked sendable types

### PR DESCRIPTION
**Pull request by Genaro-Chris**
This allow custom c++ types to be imported as sendable types if the developer manually checks and ensures that such type when used in async context would be free from all data race bugs.

### CODE REVIEW

**Comment by Genaro-Chris** [comment]
@swift-ci Please test

**Comment by hyp** [comment]
@swift-ci Please test

File: lib/ClangImporter/bridging [hunk 1]
```diff
@@ -140,6 +140,19 @@
...
+/// use safely across async contexts.
+///
+/// For example 
+/// ```
+///   class SWIFT_UNCHECKED_SENDABLE CustomUserType
+///   { ... } 
+/// ``` 
+/// Will be imported as `struct CustomUserType: @unchecked Sendable`
+#define SWIFT_UNCHECKED_SENDABLE \
+  __attribute__((swift_attr("@Sendable)))
```
Comment (hyp) [inline lib/ClangImporter/bridging:154]: you're missing a quote here

Comment (Genaro-Chris) [reply lib/ClangImporter/bridging]: Done

Comment (Genaro-Chris) [reply lib/ClangImporter/bridging]: Fixed the typo

**Comment by Genaro-Chris** [comment]
@swift-ci please smoke test

**Comment by Genaro-Chris** [comment]
Please merge

**Comment by hyp** [comment]
@Genaro-Chris please add a test case , see `test/Interop/Cxx/ergonomics/swift-bridging-annotations.swift`

**Comment by Genaro-Chris** [comment]
@swift-ci please smoke test

**Comment by hyp** [comment]
@swift-ci please test

**Review by hyp — APPROVED** [review]

**Comment by Genaro-Chris** [comment]
@swift-ci please merge

**Comment by Genaro-Chris** [comment]
@swift-ci test

**Comment by hyp** [comment]
@swift-ci please test

**Review by hyp — APPROVED** [review]

**Comment by Genaro-Chris** [comment]
@swift-ci please smoke test

**Pull request merged**